libr/debug/debug.c

```diff
@@ -68,12 +68,13 @@ static int r_debug_bp_hit(RDebug *dbg, RRegItem *pc_ri, ut64 pc, RBreakpointItem
 		return true;
 	}
 
+	/* The MIPS ptrace has a different behaviour */
+# if !(__MIPS__ || __mips__)
```

Comment:
better use positive conditionals: if (__MIPS__ || __mips__) ... also i think that lowercase should be enough (and swap then/else blocks)